scripts/apps/authoring/authoring/directives/ArticleEditDirective.ts

```diff
@@ -99,6 +99,12 @@ export function ArticleEditDirective(
                     superdesk.intent('preview', 'item', item);
                 };
 
+                elem.on('drop dragdrop', (event) => {
```

Comment:
Which element does `elem` refer to?

Reply:
@tomaskikutis Sorry, I totally missed this. It's there to reject all the items dropped when the `editable` is set to `false`. What do you mean by which element?

Reply:
You are adding an event listener on an element/DOM node. I was asking which element is that. Article edit view is made up of multiple fields - headline, body, media gallery etc. I see you are most likely adding the event listener on a root element wrapping the entire article edit view, which is fine. Though you should probably add the code removing the event listener when the view is unmounted.